python/dgl/_ffi/_cython/ndarray.pxi

```diff
@@ -1,4 +1,5 @@
 from ..runtime_ctypes import DGLArrayHandle as PyDGLArrayHandle
+from cpython cimport PyCapsule_Destructor
 
 cdef const char* _c_str_dltensor = "dltensor"
 cdef const char* _c_str_used_dltensor = "used_dltensor"
```

Comment:
We may also remove the void from L8, I guess.

Reply:
OK, I will try.